data/json/itemgroups/Weapons_Mods_Ammo/guns.json

```diff
@@ -645,7 +648,8 @@
       { "item": "axmc", "prob": 10, "charges-min": 0, "charges-max": 0 },
       { "group": "nested_ar15_with_kits", "prob": 15 },
       { "group": "nested_mdrx_with_kits", "prob": 10 },
-      { "group": "nested_axmc_with_kits", "prob": 10 }
+      { "group": "nested_axmc_with_kits", "prob": 10 },
+      { "item": "rfb_308", "prob": 10, "charges-min": 0, "charges-max": 0 }
```

Comment:
I don't think copying probability here is a great choice, but every other entry on this list has that issue...

Reply:
> but every other entry on this list has that issue

Thats the problem, yes. Or maybe idea by design, so i just use what we have at the moment.